src/scss/abstracts/_mixins.scss

```diff
@@ -1,4 +1,35 @@
 // Mixins.
+@mixin uio-focus-style($fColor, $bColor) {
+	a:focus,
+	a:hover,
+	#defaultContainer header .site-nav-wrapper .site-nav nav a:focus,
+	#defaultContainer header .site-nav-wrapper .site-nav nav a:hover,
+	#defaultContainer .content-wrapper .tags-info .tags a:focus,
+	#defaultContainer .content-wrapper .tags-info .tags a:hover,
+	#search-form input,
+	.card:focus,
+	.card:hover,
+	button:focus,
+	button:hover,
+	li .pagination-link,
+	li .pagination-link::before {
+		box-shadow: inset 0 0 0 rem(2) $fColor;
+	}
+
+	#defaultContainer .content-wrapper main .pagination .pagination-list li a[aria-current="page"] {
+		background-color: $fColor !important; // Enactors.css in infusion uses !important on this element
+		box-shadow: none;
+		color: $bColor !important; // Enactors.css in infusion uses !important on this element
+	}
+
+	#defaultContainer .content-wrapper main .pagination .pagination-list li a[aria-current="page"]:hover,
```

Comment:
This focus/hover styling is for the same element above at line 19. Can you move it in there in the format of: 
```
	#defaultContainer .content-wrapper main .pagination .pagination-list li a[aria-current="page"] {
		// regular styles

		&:hover,
		&:focus {
			// focus/hover styles
	}
```

Thanks.